faux-mgs/src/main.rs

```diff
@@ -454,6 +454,11 @@ enum Command {
         #[clap(subcommand)]
         cmd: DumpCommand,
     },
+    /// Read Host flash at address
+    ReadHostFlash {
+        #[clap(value_parser = parse_int::parse::<u32>)]
```

Comment:
mmmmaybe worth a comment indicating that this can be given in hexadecimal as well as in decimal? (that would also answer my question of "why do we use a crate to parse this rather than using Clap's built in parser for integers")

Reply:
I was kind of amazed that this was the first time in `faux-mgs` we wanted to be able to give hex addresses